js/text_encoding_test.ts

```diff
@@ -25,17 +25,6 @@ test(function btoaFailed() {
   assertEqual(err.name, "InvalidInput");
 });
 
-test(function textDecoder() {
```

Comment:
Why does this test need to go? It no longer passes?

Reply:
The issue is that this file isn't valid UTF-8, it dies on `v8::String::NewFromUtf8`... I'm not sure how it worked before. I will investigate...

Reply:
Currently code is loaded through the CodeFetch op. The source code is passed through the flatbuffer message and decoded by the flatbuffer library on the JS side. Looking at that source code, it does not die on invalid utf-8 input:
https://github.com/denoland/deno_third_party/blob/de4fbe4edfdabfbd053a0cca1bb113cfc97da73b/node_modules/flatbuffers/js/flatbuffers.js#L1082-L1138

With this change, source code is loaded more directly in C++ using `v8::String::NewFromUtf8`, which appear to be stricter.

There's no obvious solution to this.... I could run the source code through some sort of sanitizer first?

Reply:
I will land with the tests removed. The file isn't valid utf8 so its of questionable validity. 

That said, I think we ought to sanitize input and accept invalid utf8 - but it's a much more minor concern than this refactor.

Landing now.